client/interfaces/components/AddSponsor.tsx

```diff
@@ -1,7 +1,4 @@
-export interface IAddSponsorProps {
-  eventId: string;
-  chapterId: string;
-}
+// export interface IAddSponsorProps {}
 
```

Comment:
Maybe better to collocate this with the component

Reply:
Are you suggesting to keep interfaces, side-by-side to respective files, where they get referenced?

Reply:
Maybe a interfaces folder insade of components? Or .d.ts file insde of components folder? Or maybe keep it in the same file? If I think about it this actually seems like a quite elegant solution, so keep it like that maybe?

Reply:
Moved it to `/client/interfaces`, which is recreating Response types from `/types/models.d.ts` (represents DB models). Responses may or may not differ, but in terms of `interface` naming conventions, that we will be following in this project, this change now looks good.